capmap-config.js

```diff
@@ -27,6 +27,7 @@ var alertOptions = {
     iconWidth: 30,
     iconHeight: 30,
     transparentIcons: false,
+    customIcons: true,
```

Comment:
I'd suggest that you should either remove this variable from the "default" configuration file or set it to false, at the moment user gets by broken images by default and it might be confusing.

![Uploading Screenshot 2023-08-07 at 8.26.14.png…]()